C/c4Database.cc

```diff
@@ -153,7 +153,12 @@ static C4DatabaseConfig newToOldConfig(const C4DatabaseConfig2 &config2) {
 
 /*static*/ void C4Database::copyNamed(slice sourcePath, slice destinationName, const Config &config) {
     ensureConfigDirExists(config);
-    FilePath from(sourcePath);
+    std::string sourcePathString = sourcePath.asString();
```

Comment:
What about `config.parentDirectory`?  Does it need a trailing '/'?  Does it get one?

Reply:
from the tests Core has seems that the utils that call on `parentDir` (exists and delete) are not bothered if there is/not a trailing "/".

Reply:
But `config.parentDirectory` does end with `/` when set at creation.

Reply:
I think @bmeike has a point her to check. What if the parentDirectory doesn't have a trailing '/', has LiteCore handled this already?

Reply:
parentDir is find w/ or w/o trailing slash, ::mkdir() does not care of the trailing slash. mkdir A == mkdir A/

Reply:
as Jianmin confirmed, indeed it doesn't make a difference. By default config parent is taken with "/" and in the case it is manually set, I have checked for all cases where it is used and tests pass even without "/"